### PR TITLE
feat(database): separate files to support kanban view

### PR DESCRIPTION
**Pull request by zqran**
Main directory structure:

```html
/database-block
  /kanban
    kanban-view.ts
    ...
  /table
    /components
    table-view.ts
    ...
  database-block.ts
  database-model.ts
  ...
```

part of: #2401

### CODE REVIEW

File: packages/blocks/src/database-block/kanban/kanban-block.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// related component
```
Comment (doodlewind) [inline packages/blocks/src/database-block/kanban/kanban-block.ts:1]: Should be `kanban-view.ts` and `table-view.ts`

**Review by doodlewind — APPROVED** [review]

**Comment by doodlewind** [comment]
Any link for quick preview?

**Comment by doodlewind** [comment]
Got it, it's still under headless state, which is fine. Just remember to add feature flag for the slash menu!

**Comment by zqran** [comment]
> Got it, it's still under headless state, which is fine. Just remember to add feature flag for the slash menu!

Get!

**Pull request merged**